recipe/meta.yaml

```diff
@@ -43,8 +43,6 @@ source:
 
 build:
   number: {{ intel_build_number|int + dst_build_number|int }}
-  binary_relocation: false
-  detect_binary_files_with_prefix: false
   skip: True                                  # [not (linux64 or win or osx)]
```

Comment:
I think this compiler only supports x86/x64.  So the skip is insufficent.  Please rewrite condition as something like this '[x86 and (linux or win or osx)]'.  This should cover both CPU variants.